queries/v/indents.scm

```diff
@@ -0,0 +1,18 @@
+[(import_declaration
+  (const_declaration)
+  (var_declaration)
+  (type_declaration)
```

Comment:
![image](https://user-images.githubusercontent.com/7189118/155191100-3262b25a-0d31-45eb-bef6-159e5e00b05f.png)

 `var_declaration` apparently does not exist as a node

Reply:
Uh most overlooked it. Thanks